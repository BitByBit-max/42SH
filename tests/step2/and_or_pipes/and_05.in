echo "This is the first command" | cat  && echo "This is the second command" | cat
