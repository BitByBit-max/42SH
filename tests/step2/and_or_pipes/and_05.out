This is the first command
This is the second command
